assertion/assertion.mbt

```diff
@@ -69,3 +69,51 @@ test "assert_eq.eq" {
   assert_eq(1, 1)?
   assert_eq("123", "123")?
 }
+
+pub fn assert_is[T : Debug](a : T, b : T) -> Result[Unit, String] {
+  if a === b {
+    Ok(())
+  } else {
+    let a = debug_string(a)
+    let b = debug_string(b)
+    Err("assertion failed for `\(a) === \(b)`")
+  }
+}
+
+test "assert_is.is" {
+  let x = 1
+  let s = "x:\(x)"
+  assert_eq(assert_is(s, s), Ok(()))?
+}
+
+test "assert_is.is.not" {
+  let x = 1
+  let s1 = "x:\(x)"
+  let s2 = "x:\(x)"
+  assert_eq(assert_eq(s1, s2), Ok(()))?
+  assert_ne(assert_is(s1, s2), Ok(()))?
+}
+
+pub fn assert_is_not[T : Debug](a : T, b : T) -> Result[Unit, String] {
```

Comment:
Could you add some documentation comments for `assert_is` and `assert_is_not`, since their functionality isn't clear from their names?

Reply:
Yeah sure, one sec

Reply:
Now added. Thanks!